packages/react-dom/src/client/ReactDOMLegacy.js

```diff
@@ -364,15 +364,15 @@ export function unstable_renderSubtreeIntoContainer(
   if (disableLegacyMode) {
     if (__DEV__) {
       console.error(
-        'ReactDOM.unstable_renderSubtreeIntoContainer() is no longer supported in React 18. Consider using a portal instead.',
+        'ReactDOM.unstable_renderSubtreeIntoContainer() was removed in React 19. Consider using a portal instead.',
```

Comment:
This might be a fine intermediate step but for builds that no longer support legacy mode we should make sure we remove these APIs from the entrypoint itself so this error will never be seen.